README.md

```diff
@@ -927,6 +927,125 @@ interface INPAttribution {
    * are detect, this array will be empty.
    */
   longAnimationFrameEntries: PerformanceLongAnimationFrameTiming[];
+  /**
+   * If the browser supports the Long Animation Frame API, this object
+   * summarises information relevant to INP across the long animation frames
+   * intersecting the INP interaction. See the LongAnimationFrameSummary
+   * definition for an explanation of what is included.
+   */
+  longAnimationFrameSummary?: LongAnimationFrameSummary;
+    {
+      /**
+       * The number of Long Animation Frame scripts that intersect the INP
+       * interaction.
+       * NOTE: This may be be less than the total count of scripts in the Long
+       * Animation Frames as some scripts may occur before the interaction.
+       */
+      numIntersectingScripts: number;
+      /**
+       * The number of Long Animation Frames intersecting the INP interaction.
+       */
+      numLongAnimationFrames: number;
+      /**
+       * The slowest Long Animation Frame script that intersects the INP
+       * interaction.
+       */
+      slowestScript: SlowestScriptSummary;
```

Comment:
Also, I wonder if `longestScript` would be better and more consistent with other API names (e.g. Long Tasks and Long Animation Frames).